elliptic-curve/src/lib.rs

```diff
@@ -113,6 +113,7 @@ pub use zeroize;
 pub use {
     crate::{
         arithmetic::{CurveArithmetic, PrimeCurveArithmetic},
+        point::NonIdentity,
```

Comment:
FYI, it's already re-exported, just as `elliptic_curve::point::NonIdentity` (which I did in #1186): https://github.com/RustCrypto/traits/blob/master/elliptic-curve/src/point.rs#L7

It can potentially be re-exported in both places, although I think `point::NonIdentity` is a bit clearer personally.

Reply:
Ah, I forgot about #1186.
So the problem is that `mod point` isn't public, so it's still not re-exported:
https://github.com/RustCrypto/traits/blob/778e6f4368422f87ba5d667095673423f8abccda/elliptic-curve/src/lib.rs#L86
I'm happy to remove the new re-export and change `mod point` to `pub mod point`.

Reply:
Oh whoops. Yeah, I think it's about time we consider making the `point` module public.

Reply:
Done, let me know if you want me to adjust anything else.